docs/releases/1.10.txt

```diff
@@ -919,6 +919,10 @@ Miscellaneous
   To adapt, move the fragment outside the template tag:
   ``{% static 'img.svg' %}#fragment``.
 
+* When :setting:`USE_L10N` is ``True``, localization is applied for the
```

Comment:
This could do a better job of explaining how the new behavior differs from the old behavior.